xcvm/cosmwasm/contracts/interpreter/examples/schema.rs

```diff
@@ -0,0 +1,15 @@
+use cosmwasm_schema::{export_schema, remove_schemas, schema_for};
+use std::env::current_dir;
+use std::fs::create_dir_all;
+
+use xcvm_interpreter::msg::{ExecuteMsg, InstantiateMsg};
+
+fn main() {
+    let mut out_dir = current_dir().unwrap();
+    out_dir.push("schema");
+    create_dir_all(&out_dir).unwrap();
+    remove_schemas(&out_dir).unwrap();
+
+    export_schema(&schema_for!(InstantiateMsg), &out_dir);
+    export_schema(&schema_for!(ExecuteMsg), &out_dir);
+}
```

Comment:
Same, why in `example` dir?

Reply:
[see](https://github.com/ComposableFi/composable/pull/1436#discussion_r943152450)